test/startservers.py

```diff
@@ -139,6 +145,42 @@ def check():
     processes = stillok
     return not busted
 
+def startChallSrv():
+    """
+    Start the pebble-challtestsrv and wait for it to become available. See also
+    stopChallSrv.
+    """
+    global challSrvProcess
+    if challSrvProcess is not None:
+        return
```

Comment:
Can we make this an exception? I think we expect this to only be called once.

Reply:
Good idea, thanks! Fixed in 19d0988